### PR TITLE
Post Camel 3.2.0 upgrade fixes

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

File: poms/bom/pom.xml [hunk 1]
```diff
@@ -2069,16 +2069,16 @@
...
             </dependency>
-            <dependency>
-                <groupId>org.yaml</groupId>
-                <artifactId>snakeyaml</artifactId>
-                <version>${snakeyaml.version}</version>
-            </dependency>
             <dependency>
                 <groupId>org.threeten</groupId>
                 <artifactId>threetenbp</artifactId>
                 <version>1.4.0</version>
```
Comment (lburgazzoli) [inline poms/bom/pom.xml:2075]: we should probably replace versions with properties

Comment (jamesnetherton) [reply poms/bom/pom.xml:2075]: Yes, I'll do that. Thought we had some Groovy hackery which checked that stuff...

Comment (lburgazzoli) [reply poms/bom/pom.xml:2075]: IIRC, not at the pom level but only at extension/integration test

File: extensions-jvm/pom.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@
         <module>aws2-translate</module>
         <module>cassandraql</module>
         <module>couchbase</module>
-        <module>debezium-mongodb</module>
+        <!--module>debezium-mongodb</module-->
```
Comment (ppalaga) [inline extensions-jvm/pom.xml:54]: Would you please file a followup issue?

File: extensions/pom.xml [hunk 3]
```diff
@@ -98,8 +98,7 @@
         <module>mail</module>
         <module>microprofile-health</module>
         <module>microprofile-metrics</module>
-<!--        Disable MongoDB until quarkus uses mongbd 4.x -->
-<!--        <module>mongodb</module>-->
+        <!--        <module>mongodb</module>-->
```
Comment (ppalaga) [inline extensions/pom.xml:101]: Would you please file a followup issue? I think one is enough for all mongo extensions.

File: poms/bom/pom.xml (hunk 1 above)
Comment (ppalaga) [reply poms/bom/pom.xml]: +1 for the prop.

File: extensions/pom.xml (hunk 3 above)
Comment (jamesnetherton) [reply extensions/pom.xml:101]: https://github.com/apache/camel-quarkus/issues/1064

File: extensions-jvm/pom.xml (hunk 2 above)
Comment (lburgazzoli) [reply extensions-jvm/pom.xml:54]: https://github.com/apache/camel-quarkus/issues/1064

**Pull request merged**